installer/sharch_body.sh

```diff
@@ -31,7 +31,8 @@ export cur_wd
 archive_path=$(realpath "$0")
 tmp_dir=$(mktemp -d)
 if [ "$(id -u)" = "0" ] ; then
-    mount -t tmpfs tmpfs-installer $tmp_dir || exit 1
+    mount_size=$(df $tmp_dir | tail -1 | tr -s ' ' | cut -d' ' -f4)
```

Comment:
What is this logic for? Are you trying to take available space from tmpfs??

Reply:
yes

Reply:
@antony-rheneus  Updated mount size depends on image size.